content/identity-service-api.md

```diff
@@ -38,8 +38,20 @@ Appendix.
 
 The mandatory baseline for identity server communication in Matrix is
 exchanging JSON objects over HTTP APIs. HTTPS is required for
-communication, and all API calls use a Content-Type of
-`application/json`. In addition, strings MUST be encoded as UTF-8.
+communication.
+
+All `POST` and `PUT` endpoints, with the exception of [`POST
+/_matrix/identity/v2/account/logout`](#post_matrixidentityv2accountlogout),
```

Comment:
there's always an exception.

Reply:
Why is `/account/logout` an exception to this?

Reply:
I suspect it's because nobody thought about it before, but: it just is, according to both the openapi definition, and the `sydent` implementation.

Reply:
(and https://spec.matrix.org/v1.3/identity-service-api/#authentication, which says "no ... request body")

Reply:
It might be nice to put a "for historical reasons" somewhere in that sentence (though I'm not sure where it would fit best) to indicate why it's like that, and that it isn't because that endpoint is actually special (as opposed to the other exceptions (media upload/download) which have actual reasons for not being JSON).  But if there's no nice way to fit that in, it's not the end of the world.

Reply:
```suggestion
All `POST` and `PUT` endpoints, with the exception (for historical reasons) of [`POST
/_matrix/identity/v2/account/logout`](#post_matrixidentityv2accountlogout),
```